src/ListVideos.jsx

```diff
@@ -0,0 +1,34 @@
+import React, {useState}from "react";
+import {VideoItems, Parag, TitleVideo, Secondary, IMG} from './components/VideoPlayer.js';
+import { useHistory  } from "react-router-dom";
+
+
+
+function ListVideos({title, description,url, videoId, videoList, setVideoData}) {
+       
+
+      let history = useHistory();
+      function  playVideo(event) {
+        history.push(`/${videoId}`);
+        console.log(title);
```

Comment:
I recommend removing all the console.logs from all your files before the PR creation